pkg/exporter/cmd/config.go

```diff
@@ -16,7 +16,8 @@ type InspServerConfig struct {
 }
 
 type MetricsConfig struct {
-	Probes []ProbeConfig `yaml:"probes" mapstructure:"probes" json:"probes"`
+	Probes           []ProbeConfig `yaml:"probes" mapstructure:"probes" json:"probes"`
+	AdditionalLabels string        `yaml:"additionalLabels" mapstructure:"additionalLabels" json:"additionalLabels"`
```

Comment:
AdditionalLabels  should be a string list?

```
metrics:
  additionalLabels:
  - name1=value1
  - name2=${labels:app}
  - name3=prefix_${labels:owner}_suffix
```

Reply:
According to the format in the issue specification, it is indeed a string. Of course, adding '-' in front is an array. Do I to modify it to add '-' in front